sal_interfaces/LinearStage/LinearStage_Commands.xml

```diff
@@ -92,16 +92,16 @@
 		<Explanation>http://sal.lsst.org</Explanation>
 		<item>
 			<EFDB_Name>stopped</EFDB_Name>
```

Comment:
called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."